index.js

```diff
@@ -0,0 +1,26 @@
+const hamburger = document.querySelector('.menu-button');
```

Comment:
- [ ] You have done a good job with the mobile menu :fire: but there is an issue because when I click a link or the (X) to close the menu it works fine the first time but after that, I am not able to open the menu again. Please, kindly fix this issue for a better user experience :wink: :pushpin: